src/legacy/core_plugins/kibana/index.js

```diff
@@ -129,12 +129,100 @@ export default function (kibana) {
       ],
 
       savedObjectSchemas: {
+        'index-pattern': {
+          icon: 'indexPatternApp',
+          titleSearchField: 'title',
+          getTitle(obj) {
+            return obj.attributes.title;
+          },
+          getEditUrl(obj) {
+            return `/management/kibana/index_patterns/${obj.id}`;
+          },
+          getInAppUrl(obj) {
+            return {
+              path: `/app/kibana#/management/kibana/index_patterns/${obj.id}`,
+              uiCapabilitiesPath: 'management.kibana.index_patterns',
+            };
+          },
+        },
+        visualization: {
+          icon: 'visualizeApp',
+          titleSearchField: 'title',
+          getTitle(obj) {
+            return obj.attributes.title;
+          },
+          getEditUrl(obj) {
+            return `/management/kibana/objects/savedVisualizations/${obj.id}`;
+          },
+          getInAppUrl(obj) {
+            return {
+              path: `/app/kibana#/visualize/edit/${obj.id}`,
+              uiCapabilitiesPath: 'visualize.show',
+            };
+          },
+        },
+        search: {
+          icon: 'search',
+          titleSearchField: 'title',
+          getTitle(obj) {
+            return obj.attributes.title;
+          },
+          getEditUrl(obj) {
+            return `/management/kibana/objects/savedSearches/${obj.id}`;
+          },
+          getInAppUrl(obj) {
+            return {
+              path: `/app/kibana#/discover/${obj.id}`,
+              uiCapabilitiesPath: 'discover.show',
+            };
+          },
+        },
+        dashboard: {
+          icon: 'dashboardApp',
+          titleSearchField: 'title',
+          getTitle(obj) {
+            return obj.attributes.title;
+          },
+          getEditUrl(obj) {
+            return `/management/kibana/objects/savedDashboards/${obj.id}`;
+          },
+          getInAppUrl(obj) {
+            return {
+              path: `/app/kibana#/dashboard/${obj.id}`,
+              uiCapabilitiesPath: 'dashboard.show',
+            };
+          },
+        },
+        url: {
+          titleSearchField: 'url',
+          getTitle(obj) {
+            return obj.attributes.url;
+          },
+        },
         'kql-telemetry': {
           isNamespaceAgnostic: true,
         },
         'sample-data-telemetry': {
           isNamespaceAgnostic: true,
+          isImportableAndExportable: false,
+        },
+        server: {
+          isImportableAndExportable: false,
+        },
+        'kql-telemetry': {
+          isImportableAndExportable: false,
         },
+        config: {
+          getInAppUrl() {
```

Comment:
`config` type should redirect to `Advanced Settings` but missing `show` permission.